resource_processor/vmss_porter/runner.py

```diff
@@ -212,29 +228,43 @@ def service_bus_message_generator(sb_message, status, deployment_message, output
     return resource_request_message
 
 
-async def deploy_porter_bundle(msg_body, sb_client, env_vars, message_logger_adapter):
+async def invoke_porter_action(msg_body, sb_client, env_vars, message_logger_adapter) -> bool:
+    """
+    Handle resource message by invoking specified porter action (i.e. install, uninstall)
+    """
     installation_id = get_installation_id(msg_body)
-    job_type = "Deployment" if msg_body['action'] == "install" else "Deleting"
-    message_logger_adapter.info(f"{installation_id}: {job_type} job configuration starting")
+    action = msg_body["action"]
+    message_logger_adapter.info(f"{installation_id}: {action} action configuration starting")
     sb_sender = sb_client.get_queue_sender(queue_name=env_vars["deployment_status_queue"])
-    if job_type == "Deployment":
+
+    # If the action is install, post message on sb queue to start a deployment job
+    if action == "install":
         resource_request_message = service_bus_message_generator(msg_body, strings.RESOURCE_STATUS_DEPLOYING, "Deployment job starting")
         await sb_sender.send_messages(ServiceBusMessage(body=resource_request_message, correlation_id=msg_body["id"]))
-    porter_command = await build_porter_command(msg_body, env_vars)
+
+    # Build and run porter command (flagging if its a built-in action or custom so we can adapt porter command appropriately)
+    is_custom_action = False if action in ["install", "upgrade", "uninstall"] else True
```

Comment:
think this can be simplified to...

`is_custom_action = action not in ["install", "upgrade", "uninstall"]`

Reply:
Right you are :) thanks